### PR TITLE
adding services to list of custom left sidebar menu pages

### PR DESCRIPTION
**Pull request by kaylima**
## Changes

Added the Services as a page that will have a left sidebar menu of child pages. Cleaned up logic for showing menu.

## Why

The Services section of inn.org is new and comprised of many different related pages. It needs its' own menu, similar to the current About page.

## Questions

@benlk How does the switch/case look? And, how should we test?

### CODE REVIEW

File: single-one-column.php [hunk 1]
```diff
@@ -32,27 +33,38 @@
...
 		echo '<div class="internal-subnav span3 visible-desktop">';
 	}
 
-	// about and member pages and children get their respective page trees
-	if ( $show_menu == 'About' || $show_menu == 'Membership' ) {
-		$pg_id = ( $show_menu == 'About' ) ? $about_pg_id : $members_pg_id;
+	// about, member and services pages and children get their respective page trees
+	switch ( $show_menu) {
+		case "About":
+			$pg_id = $about_pg_id;
```
Comment (benlk) [inline single-one-column.php]: These cases need `break` statements; I'll add them.

Comment (kaylima) [reply single-one-column.php]: Ah yes, thanks for catching that!

Comment (kaylima) [reply single-one-column.php]: https://github.com/INN/inn/pull/96/commits/ea51da4f674facbd6ee10b4ecda981714c50fe3b (if you haven't already)

**Comment by benlk** [comment]
Updated with various code quality fixes and bug fixes in this file.

This is based off of the current "master" branch, which is *not* what's currently deployed to inn.org. The deployed branch appears to be https://github.com/INN/inn/tree/tyler-style-refresh from https://github.com/INN/inn/pull/93, at commit https://github.com/INN/inn/pull/93/commits/c7c69cdd503a57ce6c5262b3d3a89979e9c4dbbf from 28 days ago.

This code is deployable, but before deploying this branch, we need to discuss:

- [ ] is https://github.com/INN/inn/pull/93 ready to be merged into master? 
    - [ ] If so:
        1. merge that into master
        2. then merge this into master.
        3. deploy master as part of umbrella-inndev's `master` branch.
    - [ ] if not:
        1. merge https://github.com/INN/inn/pull/93/commits/c7c69cdd503a57ce6c5262b3d3a89979e9c4dbbf into master,
        2. merge master into this branch
        3. merge this branch into master
        4. deploy master as part of umbrella-inndev's `master` branch.
        5. work on `tyler-style-refresh` continues in https://github.com/INN/inn/pull/93
- [ ] how do we prevent this snafu with half-deployed branches again?

**Comment by benlk** [comment]
Have now merged #93 into master, and master into this branch.

**Comment by benlk** [comment]
Oh, this is merging into *staging*

**Comment by benlk** [comment]
Have now merged master into staging: a fast-forward merge that brought staging up to match master. This PR now only reflects the differences made in this PR, and not the differences from #93 or previous work on master since staging was last merged into master.

**Pull request merged**